### ISSUE TITLE
Autosave on new page form creates multiple new pages

### ISSUE THREAD

**Issue by timoludwig** [Integreat/integreat-cms#672]
### Describe the Bug

If the new page form is opened, every 2 minutes a new page is created by the autosave functionality. We need to somehow keep track of the first autosave and update the existing draft afterwards instead of creating new ones.

### Steps to Reproduce

1. Go to a region
2. Create new page
3. Type in a valid title
4. Wait at least 2 minutes
5. See new page draft in the page tree

Additionally, the form gets also submitted when the user leaves the page without saving.

### Expected Behavior

Autosaves of new pages should update the same draft and not create new instances every 2 minutes

### Actual Behavior

Every 2 minutes a new page is created

### Additional Information

If the expected behavior is too hard to implement, we could also deactivate autosave on the page creation form.

This issue depends on:
- Issue #869
- Issue #870

**Comment by MizukiTemma** [comment]
Here too I tried reproduce of the bag and found some stuffs.

In the text editor of a published page with title "inital" and text "inital, I added a new word "asdf" in the text.
After an auto save, the text change was not shown in the console, but by clicking "save as draft" the change appered in the console.

So I would like to suggest deactivating the autosave (because it does not reflect the change) and saving the contents in the same way with an explicit "save as draft" when the use leaves the editor.

*comment on the screen shot: the first two marked are from auto save, the third is from click on "save as draft"
![Inkedautosave1_LI](https://user-images.githubusercontent.com/82345046/122192783-4307d300-ce94-11eb-8470-c95982d25df2.jpg)

**Comment by timoludwig** [comment]
@MizukiTemma I'm sorry it took me so long to get back to your question!

In general, we cannot deactivate autosave completely because our clients need this feature. Thus, we have to find out why it's not working and fix it.

The first steps are:
- For easier testing, I would recommend to set a lower `autosave_interval` in [src/frontend/js/forms/tinymce-init.ts](https://github.com/Integreat/integreat-cms/blob/5d3a30ad71108662c3f45ce8674a71096bdf59f7/src/frontend/js/forms/tinymce-init.ts#L89) so you don't have to wait 2 minutes each time you want to test the autosave feature
- Find out why the autosave is not actually saved. Looking at the response of the autosave ajax call, we can find the message "status: This field is required.", because we set the status via the submit buttons and the autosave does not include the submit buttons. So as a quick fix, I would suggest sending a post status in the autosave ajax function in [src/frontend/js/forms/autosave.ts](https://github.com/Integreat/integreat-cms/blob/develop/src/frontend/js/forms/autosave.ts), e.g. by adding the following line:
  ```
  let formData = new FormData(form);
  formData.append("submit_draft", "true");
  ```
  Now, the draft is actually saved and the buggy behavior of this issue is reproduced.
- Then, we have to find out why the "text" field is not submitted, only the title, slug and other fields... I think this might be because tinymce seems to submit its value into the html textarea field only when the form is submitted via a submit button, not when serialized via FormData. A solution is described e.g. [here](https://stackoverflow.com/questions/2122085/jquery-and-tinymce-textarea-value-doesnt-submit): Add `import tinymce from "tinymce";` to [src/frontend/js/forms/autosave.ts](https://github.com/Integreat/integreat-cms/blob/develop/src/frontend/js/forms/autosave.ts) and insert `tinymce.triggerSave();` before `let formData = new FormData(form);`.
- Then, we can think about solving the issue described in the original bug report. One possibility I see is that after the very first autosave, we change the action of the `#content_form` to the url which is returned by the ajax call. When we submit data to the url `http://localhost:8000/augsburg/pages/de/new`, it will redirect to the new url of the new created page, e.g. `http://localhost:8000//augsburg/pages/de/81/edit`. If we now replace the form action and use the form action for future ajax calls instead of `window.location.href` [here](https://github.com/Integreat/integreat-cms/blob/5d3a30ad71108662c3f45ce8674a71096bdf59f7/src/frontend/js/forms/autosave.ts#L6), the page will be updated in the future instead of creating new page instances every time. What do you think about this solution? Does it makes sense to you?

**Comment by timoludwig** [comment]
I created the new issues #869 and #870 for the new bugs you found. These issues have to be fixed before #672 and #714.
